dsgrid/dataset/growth_rates.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+import pyspark.sql.functions as F
+from pyspark.sql.types import IntegerType
+
+from dsgrid.dimension.base_models import DimensionType
+from dsgrid.exceptions import DSGInvalidQuery
+from dsgrid.query.models import ExponentialGrowthDatasetModel
+from dsgrid.utils.spark import get_unique_values
+
+
+logger = logging.getLogger(__name__)
+
+
+def apply_growth_rate_123(
```

Comment:
@lixiliu The names here are obviously bogus. How would you like to deal with them? Would you like to takeover and create your own structure of name/function mappings?

Reply:
here or in new PR?

Reply:
For now we could call this one apply_annual_growth_rate?

Reply:
If @lixiliu is going to implement this soon, I would prefer to do nothing.